content/src/fieldConfig/profile.json

```diff
@@ -266,7 +266,7 @@
       },
       "documents": {
         "default": {
-          "header": "Reference Documents",
+          "header": "",
```

Comment:
These are no longer rendered since there's a page header that says the same thing, and that better fits accessibility hierarchy